features/upgrade/storage/upgrade.feature

```diff
@@ -344,3 +344,120 @@ Feature: Storage upgrade tests
       | ls | /mnt/hostpath |
     Then the output should contain:
       | test-before-upgrade |
+
+  # @author ropatil@redhat.com
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Snapshot operator should work well before and after upgrade of cluster on AWS platform - prepare
+    Given I switch to cluster admin pseudo user
+    Given the master version >= "4.7"
+      
+    #Snapshot operator/controller status checking
+    Given the "csi-snapshot-controller" operator version matches the current cluster version
+
+    Given the status of condition "Degraded" for "csi-snapshot-controller" operator is: False
```

Comment:
@ropatil010 @duanwei33 From my side, the status of csi-snapshot-controller cluster check just need one. Check the status of `condition “Available” for “csi-snapshot-controller” operator is True` is not enough? Could you please help to explain the doubt for me? Thanks!

Reply:
These steps could make the verification more solid, making sure the cluster operator is available and does not have any unhealthy state